packages/models-library/src/models_library/api_schemas_clusters_keeper/clusters.py

```diff
@@ -20,6 +20,6 @@ class OnDemandCluster(BaseModel):
     authentication: ClusterAuthentication
     state: ClusterState
     user_id: UserID
-    wallet_id: WalletID | None
+    wallet_id: WalletID | None = None
```

Comment:
why this change? this will change the logic here.

Reply:
See: https://github.com/ITISFoundation/osparc-simcore/pull/6333#discussion_r1751389736

Reply:
@sanderegg https://github.com/pydantic/pydantic/issues/5481

Reply:
@giancarloromeo sorry but I disagree. Please see: https://docs.pydantic.dev/latest/migration/#unions
What you did here makes wallet_id None by default, which is not the wanted behavior. It should be Required but nullable

Reply:
@sanderegg please, have a look at the documentation of previous Pydantic v1:
https://docs.pydantic.dev/1.10/usage/models/#required-optional-fields